Author: Tibor Vass <tibor@docker.com>, Tianon Gravi <admwiggin@gmail.com>
Description: replace "code.google.com/p/go.net" with the canonical "golang.org/x/net"
Applied-Upstream: https://github.com/docker/docker/pull/13972 (Docker 1.8+)

Index: docker.io/api/server/server.go
===================================================================
--- docker.io.orig/api/server/server.go
+++ docker.io/api/server/server.go
@@ -13,8 +13,8 @@ import (
 	"strings"
 	"time"
 
-	"code.google.com/p/go.net/websocket"
 	"github.com/gorilla/mux"
+	"golang.org/x/net/websocket"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/api"
Index: docker.io/integration-cli/docker_api_attach_test.go
===================================================================
--- docker.io.orig/integration-cli/docker_api_attach_test.go
+++ docker.io/integration-cli/docker_api_attach_test.go
@@ -9,7 +9,7 @@ import (
 
 	"github.com/go-check/check"
 
-	"code.google.com/p/go.net/websocket"
+	"golang.org/x/net/websocket"
 )
 
 func (s *DockerSuite) TestGetContainersAttachWebsocket(c *check.C) {
